build.xml

```diff
@@ -113,6 +114,10 @@
 		</exec>
 	</target>
 
+	<target name="test-configuration-validate">
```

Comment:
@ondrejmirtes 

Wondering, would it make sense to add dependencies to individual tasks?

After all, this task actually depends on the `composer-install` task (and so do a few others).

What do you think?